FCollada/FUtils/FUFileManager.cpp

```diff
@@ -210,7 +211,7 @@ bool FUFileManager::MakeDirectory(const fstring& directory)
 	if (_mkdir(TO_STRING(absoluteDirectory).c_str()) == 0) return true;
 	errno_t err; _get_errno(&err);
 	if (err == EEXIST) return true;
-#elif defined(LINUX)
+#elif defined(LINUX) || defined(__APPLE__)
 	if (mkdir(TO_STRING(absoluteDirectory).c_str(), ~0u) == 0) return true; // I think this means all permissions..
 #elif defined(__APPLE__)
```

Comment:
Should this case not be entirely removed then, since it's now being shadowed?

Reply:
Good point. Done!